cmd/upgrade/operatorroles/cmd.go

```diff
@@ -214,12 +214,13 @@ func run(cmd *cobra.Command, argv []string) error {
 		os.Exit(1)
 	}
 
+	mode, err = handleModeFlag(cmd, skipInteractive, mode, err, r.Reporter)
```

Comment:
Do we need to wrap it with the following condition?
`if isOperatorPolicyUpgradeNeeded || len(missingRolesInCS) > 0 {`

Reply:
the condition is already checked earlier and we return if it is not satisfied

Reply:
Oh right, missed that, thanks.